website/docs/engine-flink/delta-joins.md

```diff
@@ -172,9 +172,9 @@ Refer to the [Delta Join Issue](https://issues.apache.org/jira/browse/FLINK-3783
 
 #### Limitations
 
-- The primary key or the prefix lookup key of the tables must be included as part of the equivalence conditions in the join.
+- The primary key or the prefix key of the tables must be included as part of the equivalence conditions in the join.
 - The join must be a INNER join.
-- The downstream nodes of the join can accept duplicate changes, such as a sink that provides UPSERT mode.
+- The downstream nodes of the join can accept duplicate changes, such as a sink that provides UPSERT mode without `upsertMaterialize`.
```

Comment:
It's hard to understand what is a sink without "upsertMaterialize". Could you explain more about this?

Reply:
I think it's fine to link it